src/domain/entities/noteSettings.ts

```diff
@@ -31,6 +34,11 @@ export default interface NoteSettings {
    * Invitation hash
    */
   invitationHash: InvitationHash;
+
+  /**
+   * Team members
+   */
+  team?: Team;
```

Comment:
describe the case when there is no `team` property in Note Settings. Is there any such case?

Reply:
I think this case may happen, because team object is not created automatically. Also, we don't need this field when we create new noteSettings (in sequelize/noteSettings for example)

Reply:
The main reason is that we've decided to not to store note Creator at the "team". That's why team is empty by default. Please, add this explanation to the jsdoc.

P. S. 
Bytheway, it its a subject to change. If we will store creator in a team, so:
- permissions and access control check will be simplified . We won't need a special login to handle creation and the whole permission management will be consistent
- it will be possible to change a Creator's role
We need to think about it.